packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -181,9 +181,92 @@ class _DomainCheckTable extends StatelessWidget {
           ],
         ),
         _Fingerprint(controller: controller),
-        if (linkData.domainErrors.isNotEmpty)
-          _DomainFixPanel(controller: controller),
+        _AssetLinksJsonFileIssues(controller: controller),
+        _HostingIssues(controller: controller),
         const SizedBox(height: intermediateSpacing),
+        const _ViewDeveloperGuide(),
+      ],
+    );
+  }
+}
+
+class _AssetLinksJsonFileIssues extends StatelessWidget {
+  const _AssetLinksJsonFileIssues({
+    required this.controller,
+  });
+
+  final DeepLinksController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    final errors = controller.selectedLink.value!.domainErrors
+        .where(
+          (error) => domainAssetLinksJsonFileErrors.contains(error),
+        )
+        .toList();
```

Comment:
is this expensive? should we store this set of errors on the controller so the build method does not have to pay this cost each time?

Reply:
I don't think it's expensive, the `domainErrors` has size <= 7 ,  the domainAssetLinksJsonFileErrors is a set of size 3